### PR TITLE
Testsuites polish: Better "no replays found" error

### PR DESCRIPTION
**Pull request by jonbell-lot23**
**Old:**
image

**New:**
image

### CODE REVIEW

File: src/ui/components/Library/Team/View/Tests/Overview/ReplayList.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { TestExecution } from "shared/test-suites/TestRun";
 
 import { getTruncatedRelativeDate } from "../../Recordings/RecordingListItem/RecordingListItem";
 import styles from "../../../../Library.module.css";
+import testsuiteStyles from "../../../../Testsuites.module.css";
```
[the remark this answers is not in the document]
Comment (jonbell-lot23) [reply src/ui/components/Library/Team/View/Tests/Overview/ReplayList.tsx:10]: I'll be sure to move things into a shared naming convention. We have a lot of work to do here, lots of the time we're copy/pasting the same stuff in two views even though they're part of one feature/experience.

[the remark this answers is not in the document]
Comment (bvaughn) [reply src/ui/components/Library/Team/View/Tests/Overview/ReplayList.tsx:10]: In this case, importing a CSS module 4 directories removed, I don't think "shared" in the name is sufficient.

[the remark this answers is not in the document]
Comment (jonbell-lot23) [reply src/ui/components/Library/Team/View/Tests/Overview/ReplayList.tsx:10]: We'll need to figure out a plan. We used a ton of directories but I need to be able to define a single CSS class that each of our columns/views can refer to.

The best approach I can see is to put the css at the top of testsuites, and that's going to mean searching up the tree to get it. We can't do it per directory without lots of silly re-use.

[the remark this answers is not in the document]
Comment (jonbell-lot23) [reply src/ui/components/Library/Team/View/Tests/Overview/ReplayList.tsx:10]: Just filed this: https://linear.app/replay/issue/FE-2120/add-top-level-testsuites-css

A couple thoughts that I may work through in a Notion document or something:

Bad idea 1: If I were to only work in Figma and not update any CSS ever, that's a huge bottleneck. It takes much longer to explain some things in Figma, so doing zero code and only communicating in pictures is slow and bad.

Bad idea 2: If I were to dive into code, make dumb decisions, not get reviews, and break everything, that's obviously bad too.

So the key is a good sweet spot between the two. Some things are served really well by me defining a feature in Figma and then handing off to FE. We've done this well many times. Other things are served really well by me dropping into the CSS and making the sorts of polish changes that often go unnoticed/deprioritised in a standard eng queue. We've also done this well many times.

I think for this issue, I think we found the line between the two. I did a ton of good polish work yesterday, and most of it was good/helpful. But then there are some philosophical things like this that are good to hand over to FE.

**Pull request merged**